seqr/views/apis/variant_search_api_tests.py

```diff
@@ -312,6 +312,28 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts, mock_erro
              '', '', '', '', '']]
         self.assertEqual(response.content, ('\n'.join(['\t'.join(line) for line in expected_content])+'\n').encode('utf-8'))
 
+        # test export with max families
+        with mock.patch('seqr.views.apis.variant_search_api.MAX_FAMILIES_PER_ROW', 1):
```

Comment:
It seems too extreme to set `MAX_FAMILIES_PER_ROW` to 1. Can it be 2 or 3?

Reply:
No, there are only max 2 families in the fixture days so setting it to anything over 1 doesn't do anything. Adding new fixture data would be difficult and would break many other tests, and there's no real benefit anyways. Using 1 here does just as good a job testing the behavior as using a bigger number